chart/Chart.yaml

```diff
@@ -0,0 +1,254 @@
+apiVersion: v2
+name: "magda-config"
+description: ""
+version: "0.0.57-0"
+kubeVersion: ">= 1.14.0-0"
+dependencies:
+  - name: magda-core
```

Comment:
Seeing you've removed the connectors from `magda` chart so probably you want to depend on `magda` chart instead (as it includes default minions)?